Compiler/src/verifytrim.jl

```diff
@@ -256,9 +256,28 @@ function verify_codeinstance!(interp::NativeInterpreter, codeinst::CodeInstance,
                 warn = true # downgrade must-throw calls to be only a warning
             end
         elseif isexpr(stmt, :cfunction)
+            length(stmt.args) != 5 && continue # error at run-time
+            (pointer_type, f, rt, at, call_type) = stmt.args
+
+            at isa SimpleVector || continue  # required by IR legality
+            if call_type === QuoteNode(:ccall)
+                ft = argextype(f, ci, sptypes)
+                argtypes = Any[ft]
+                for i = 1:length(at)
+                    push!(argtypes, sp_type_rewrap(at[i], get_ci_mi(codeinst), #= isreturn =# false))
+                end
+                atype = argtypes_to_type(argtypes)
+
+                mi = compileable_specialization_for_call(interp, atype)
+                if mi !== nothing
+                    # XXX: Codegen may choose unpredictably to emit this `@cfunction` as a dynamic invoke or a full
```

Comment:
```suggestion
                    # n.b.: Codegen may choose unpredictably to emit this `@cfunction` as a dynamic invoke or a full
```

I'm not really sure why you'll calling this out here, when this is exactly true of all the verification being done here.